src/main/resources/templates/db_scripts/note_data_update_metadata_user_info.sql

```diff
@@ -30,3 +34,9 @@ language 'plpgsql';
 DROP TRIGGER IF EXISTS set_note_data_md_json_trigger ON note_data CASCADE;
 
 CREATE TRIGGER set_note_data_md_json_trigger BEFORE UPDATE ON note_data   FOR EACH ROW EXECUTE PROCEDURE set_note_data_md_json();
+
+DROP TRIGGER IF EXISTS set_note_data_md_trigger ON note_data CASCADE;
+
+DROP FUNCTION IF EXISTS note_data_set_md();
```

Comment:
Similarly, "set_note_data_md_json()" is the function I see defined. Where is "note_data_set_md()"?

Reply:
@sduvvuri-ebsco we do not need  `note_data_set_md()` function, that is why we just delete it. The main reason of it was to store the creation date as a separate column.